script/tasks/diamondEMERGENCYPause.sh

```diff
@@ -88,7 +88,7 @@ function diamondEMERGENCYPause {
 
   # get user info and send message to discord server
   local USER_INFO=$(getUserInfo)
-  sendMessageToDiscordSmartContractsChannel ":warning: an emergency diamond action was just triggered (action: $ACTION, user info: $USER_INFO). Please immediately investigate if this action was not planned. :warning:"
+  sendMessageToSlackSmartContractsChannel ":warning: an emergency diamond action was just triggered (action: $ACTION, user info: $USER_INFO). Please immediately investigate if this action was not planned. :warning:"
```

Comment:
did you test this functionality?
And is the `:warning:` working for slack like it does for Discord?

Reply:
yes I did. Previousl tested externally and removed $ACTION and $USER_INFO and it works through `diamondEMERGENCYPause.sh` as well

<img width="1320" alt="image" src="https://github.com/user-attachments/assets/040726ce-eadf-4f2f-ade9-b4d226746609" />

Reply:
@0xDEnYO

Reply:
nice